include/seqan3/search/algorithm/detail/search.hpp

```diff
@@ -183,15 +180,12 @@ inline auto search_single(index_t const & index, query_t & query, configuration_
 }
 
 /*!\brief Search a query or a range of queries in an index.
- *
- * \tparam index_t   Must model seqan3::fm_index_concept.
- * \tparam queries_t Must be a std::ranges::RandomAccessRange over the index's alphabet.
- *                   a range of queries must additionally model std::ranges::ForwardRange.
- *
+ * \tparam index_t    Must model seqan3::fm_index_concept.
```

Comment:
Change to must satisfy `seqan3::fm_index_concept`

Reply:
I thought we agreed on `model` and not `satisfy` quite a while ago?

Reply:
I think this is inconsistent throughout the library...